src/js/report_form.js

```diff
@@ -1,4 +1,5 @@
 (function reportFormSetUp() {
+  const defaultDate = '1970-01-01'
```

Comment:
Why is this the default?  I get that it's the start of the epoch but why not "today"?

Reply:
it can be any random day. we just need some day because moment js cannot be initialized with just 14:00 -- it needs a full date. some of the example i saw online recommended just appending the start of the epoch to the time and then doing timezone conversion from there and parsing out the time.

for weekly we calculate the difference between the timezone shifted date and non-timezone shifted date and add the offset to the current timestamp

Reply:
OK